rundeckapp/grails-app/jobs/rundeck/quartzjobs/ExecutionJob.groovy

```diff
@@ -173,6 +175,13 @@ class ExecutionJob implements InterruptableJob {
         initMap.jobSchedulerService.afterExecution(initMap.execution.asReference(), context.mergedJobDataMap, initMap.authContext)
     }
 
+    private void markStartExecutionFailure(JobExecutionContext context) {
+        MetricRegistry metricRegistry=context.jobDetail.jobDataMap.get('metricRegistry')
+        if(metricRegistry) {
+            metricRegistry.meter(MetricRegistry.name(this.class.name, 'executionJobStartFailedMeter')).mark()
```

Comment:
hmm, using `this.class.name` will now produce two different "metrics", do you want to consolidate them?

Reply:
Yea, that would make things easier. Is there an existing pattern for that or should I just pick one class name and use it?

Reply:
perhaps just use ExecutionService class name as the basis